AUTHORS

```diff
@@ -12,3 +12,4 @@ project:
 
 Alexei Starovoitov    ast@plumgrid.com
 Brenden Blanco        bblanco@plumgrid.com
+Salvatore Orlando
```

Comment:
oh well, such a small think should not qualify me as an author.
anyway my email is salv.orlando@gmail.com